### PR TITLE
License headers

### PR DESCRIPTION
**Pull request by jonas**
I've kept the `sbt-header` separate from adding the headers in case we want to wait with adding the headers until later.

### CODE REVIEW

**Comment by suhailshergill** [comment]
+1 other than comment regd adding `sbt-header` to integration tests

File: build.sbt [hunk 1]
```diff
@@ -43,8 +47,6 @@ ScalariformKeys.preferences := ScalariformKeys.preferences.value
    .setPreference(PlaceScaladocAsterisksBeneathSecondAsterisk, true)
 
 defaultScalariformSettings
-addCommandAlias("format", ";compile:scalariformFormat;test:scalariformFormat")
```
Comment (gpoirier) [inline build.sbt]: Why remove the alias here?

Comment (gpoirier) [inline build.sbt]: Oh nevermind, I hadn't seen it being up top

**Comment by gpoirier** [comment]
@suhailshergill We don't have any integration tests for this project, and I wasn't sure we were ever need any given it's a standalone library, so there's currently no integration configuration.

**Comment by gpoirier** [comment]
+1

**Pull request merged**